tests/cp2k/cp2k_output

```diff
@@ -1197,7 +1197,7 @@
 
   Total energy:                                             -1761.41773472839827
 
- *** WARNING in qs_scf.F:542 :: SCF run NOT converged ***
+ *** SCF run converged in    47 steps ***
```

Comment:
this file is only used to check the content from parser. add `run converged` to parse by dpdata.